content-src/components/Base/Base.jsx

```diff
@@ -19,6 +19,8 @@ const PrefsButton = props => (
       className="icon icon-settings"
       onClick={props.onClick}
       data-l10n-id="newtab-settings-button"
+      // eslint-disable-next-line jsx-a11y/tabindex-no-positive
+      tabIndex="1"
```

Comment:
If the button should appear earlier in tab order, why not move the button to be earlier?

Reply:
I can do that - I wasn't sure if it was rendered at the bottom of the body for a reason. (it's a component placed at the end of the body wrapper, after sections.)

Reply:
Does the pref button need to appear earlier in tab order? It should be tabble right now as the last item.

Reply:
It is tabbable as the last item, but I don't know if that's what we want given that it's positioned to be a the top of the page - logically to me you'd be able to tab to it before reaching the rest of the content. Moving the component doesn't seem to interfere with anything else, but is there someone else we should check with before deciding?

Reply:
I've moved it to the top for now, I can always move it back if it turns out there was a reason :)

Reply:
According to Wolasi this should work fine with the prefs button rendered first, so we should be good to go!